module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -2071,13 +2075,15 @@ protected function performHoldRequest(array $requestBody): array
      * Get allowed service points for a request. Returns null if data cannot be obtained.
      *
      * @param string $instanceId  Instance UUID being requested
+     * @param string $itemId      Item UUID being requested
      * @param string $requesterId Patron UUID placing request
      * @param string $operation   Operation type (default = create)
      *
      * @return ?array
      */
     public function getAllowedServicePoints(
         string $instanceId,
+        ?string $itemId,
```

Comment:
I'm not sure why `getAllowedServicePoints` is a public method; seems like it should be protected. Maybe we should change that while we're adjusting the signature anyway.